docs/index.rst

```diff
@@ -1,26 +1,26 @@
 .. _docker-tutorial_kafka-streams-examples:
 
-Kafka Streams Demo Application
-------------------------------
+|ak| Streams Demo Application
```

Comment:
I don't think that any macros are available here... It's not like Confluent docs.

Reply:
Oh, okay I thought it went through our build system .. I'll take them out. I'm still trying to figure out how to test a build and pull in this repo, before merging.

Reply:
AFAIK, the .rst is just rendered on the GitHub page itself. Frankly, I am not even sure what the purpose of `docs/index.rst` is... for the examples repos, it seems that only the top level README file is of puspose?

Of does the docs-build somehow pull in `docs/index.rst`? For this case, the macros would be useful?

I guess @JimGalasyn should know more.

Reply:
I'm pretty sure we can use the macros here, for example we use `codewithvars::` [here ](https://github.com/confluentinc/kafka-streams-examples/blob/5.5.0-post/docs/index.rst#running-the-kafka-music-demo-application) which compiles correctly [here](https://docs.confluent.io/current/streams/kafka-streams-examples/docs/index.html#running-the-kafka-music-demo-application)

Reply:
But, I am definitely +1 on moving this to the docs repo. I had to do some hacky things just to get it to build locally a while back

Reply:
@mjsax , this file is pulled into the docs build, which is how we get this rendered version in the docs: https://docs.confluent.io/current/streams/kafka-streams-examples/docs/index.html

@ableegoldman I would love it moved to the Docs repo. @JimGalasyn , @joel-hamill what do you think?

As far as displaying on GitHub, markdown is really better for that. That said, I suppose if I _didn't_ use the substitutions, it would read a little better on GitHub.

In the meantime I am going to test the build locally to make sure it works. I think that the substitutions will work because they reference the `conf.py` file, which is where we are pulling other things like `|release|` variables, which are processed with `codewithvars`. This file is definitely being built through our Sphinx Docs build tooling.

Reply:
@londoncalling In general, I'd say it's easier for us to have everything in the main CP docs repo, if only because it makes local testing straightforward.

Reply:
Cool. Let's move this file to the docs repo and this PR can just be closed (or we use it to delete the file from this repo, after it was moved to docs?)

Reply:
@mjsax I would still like to complete the corrections to the file.

Reply:
Sorry for late reply... But I see you merge it already! Great!